web/client/plugins/DrawerMenu.jsx

```diff
@@ -15,6 +15,8 @@ const {toggleControl, setControlProperty} = require('../actions/controls');
 
 const {changeMapStyle} = require('../actions/map');
 
+const {featureClose} = require('../actions/wfsquery');
```

Comment:
Maybe the right place to trigger this action is inside the toggleMenu actionCreator, using thunk to dispatch both actions.